### PR TITLE
added examples for decision and multi bundle

### PR DESCRIPTION
**Pull request by omerlh**
Now that we have multi bundle support and the new `decision` field, it's time to update the docs.

### CODE REVIEW

File: docs/content/discovery.md [hunk 1]
```diff
@@ -149,8 +178,11 @@ Below is a policy file which generates an OPA congfiguration.
 package example
 
 discovery = {
-  "bundle": {
-    "name": bundle_name                # line 5
+  "bundles": {
+    "main": {
+      "service": "acmecorp", #line 5
```
Comment (patrick-east) [inline docs/content/discovery.md]: nit: the `line 5` comment is referenced below for the `bundle_name` variable. I think we need to update it to `line 6` now

Comment (omerlh) [reply docs/content/discovery.md]: fixed - thanks!

Comment (patrick-east) [reply docs/content/discovery.md]: Close, but we need to move the comment down to line 6 and update the paragraph below the snippet. As is its saying:

```The `bundle_name` variable in `line 5` of the above policy will be dynamically selected based on the value of the label `region`.```

Which isn't accurate for the example.

Comment (omerlh) [reply docs/content/discovery.md]: Fixed, I think it should be line 7 if I count correctly.

**Comment by tsandall** [comment]
@omerlh content-wise I think this looks fine however in #1687 the docs were reorganized and so there is a conflict now. Could you move these changes into the discovery section in the new docs/content/management.md file? Thanks!

**Comment by omerlh** [comment]
done :) another heavy git magic...

**Pull request merged**